sky/core.py

```diff
@@ -52,7 +52,7 @@ def status(all: bool, refresh: bool) -> List[Dict[str, Any]]:
         ]
 
     """
-    cluster_records = backend_utils.get_clusters(all, refresh)
+    cluster_records = backend_utils.get_clusters(True, refresh)
```

Comment:
For literal values as arguments, name them? `...(include_reserved=True, refresh=refresh)`.